NewMicroservice/NewMicroservice.xaml.cs

```diff
@@ -111,7 +112,7 @@ bool ValidateGitRepoUrl(string url)
             if (url.IsEmpty())
                 return false;
 
-            var pattern = @"^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$";
```

Comment:
When we have a username in the URL then we should be able to add URL anyway, but in the former patter it was no possible.